components/opentherm.rst

```diff
@@ -0,0 +1,363 @@
+
+- ``min_value`` (float): The minimum value. For a number this is the minimum value you are allowed to input. For an output this is the number that will be sent to the boiler when the output is at 0%.
+- ``max_value`` (float): The maximum value. For a number this is the maximum value you are allowed to input. For an output this is the number that will be sent to the boiler when the output is at 100%.
+- ``auto_max_value`` (boolean): Automatically configure the maximum value to a value reported by the boiler. Not available for all inputs.
+- ``auto_min_value`` (boolean): Automatically configure the minimum value to a value reported by the boiler. Not available for all inputs.
+
+The following inputs are available:
+
+- ``t_set``: Control setpoint: temperature setpoint for the boiler's supply water (°C)
+- ``t_set_ch2``: Control setpoint 2: temperature setpoint for the boiler's supply water on the second heating circuit (°C)
+- ``cooling_control``: Cooling control signal (%)
+- ``t_dhw_set``: Domestic hot water temperature setpoint (°C)
+- ``max_t_set``: Maximum allowable CH water setpoint (°C)
+- ``t_room_set``: Current room temperature setpoint (informational) (°C)
+- ``t_room_set_ch2``: Current room temperature setpoint on CH2 (informational) (°C)
+- ``t_room``: Current sensed room temperature (informational) (°C)
+
+Switch
+******
+
+For five status codes, switches are available to toggle them manually. The same values can be set in the hub configuration, like so:
+
+.. code-block:: yaml
+
+    opentherm:
+      ch_enable: true
+      dhw_enable: true
+
+This can be used to set the value without the need for a switch if you'd never want to toggle it after the initial configuration.
+The default values for these configuration options are listed below.
+
+For enabling of central heating and cooling, the enable-flag is only sent to the boiler if the following conditions are met:
+- the flag is set to true in the hub configuration,
+- the switch is on, if it is configured,
+- the setpoint or cooling control value is not 0, if it is configured.
+
+For domestic hot water and outside temperature compensation, only the first two conditions are necessary.
+
+The last point ensures that central heating is not enabled if no heating is requested as indicated by a setpoint of 0. If you use a number as the setpoint input and use a minimum value higher than 0, you NEED to use the ch_enable switch to turn off your central heating. In that case the flag will be set to true in the hub configuration, and setpoint is always larger than 0, so including a switch is the only way you can turn off central heating. (This also holds for cooling and CH2.)
+
+The following switches are available:
+
+- ``ch_enable``: Central Heating enabled
+- ``dhw_enable``: Domestic Hot Water enabled
+- ``cooling_enable``: Cooling enabled
+- ``otc_active``: Outside temperature compensation active
+- ``ch2_active``: Central Heating 2 active
```

Comment:
The section heading is `switches`, but then the first thing talked about is how to not use the switches. 😅

I think I would reorganize this section like so:

```suggestion

Switches are available to allow manual toggling of any of the following five status codes:

- ``ch_enable``: Central Heating enabled
- ``dhw_enable``: Domestic Hot Water enabled
- ``cooling_enable``: Cooling enabled
- ``otc_active``: Outside temperature compensation active
- ``ch2_active``: Central Heating 2 active

If you do not wish to have switches, the same values can be permanently set in the hub configuration, like so:

.. code-block:: yaml

    opentherm:
      ch_enable: true
      dhw_enable: true

This is useful when you'd never want to toggle it after the initial configuration.

The default values for these configuration variables are listed below.

For enabling of central heating and cooling, the flag is only sent to the boiler if the following conditions are met:

- the flag is set to true in the hub configuration,
- the switch is on i(f configured),
- the setpoint or cooling control value is not 0 (if configured)

For domestic hot water and outside temperature compensation, only the first two conditions are necessary.

The last point ensures that central heating is not enabled if no heating is requested as indicated by a setpoint of 0. If you use a number as the setpoint input and use a minimum value higher than 0, you **must** use the ``ch_enable`` switch to turn off your central heating. In such a case, the flag will be set to true in the hub configuration and the setpoint is always larger than 0, so including a switch is the only way you can turn off central heating. (This also holds for cooling and CH2.)
```